tests/cggi/verifier.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: heir-opt --split-input-file --verify-diagnostics %s
+// RUN: heir-opt --split-input-file --verify-diagnostics %s | FileCheck %s
```

Comment:
Actually this one doesn't need the `| FileCheck %s` - this one is using the `--verify-diagnostics` flag, and so it's not a FileCheck test, but asserting on error messages using MLIR's diagnostic tests: https://mlir.llvm.org/getting_started/TestingGuide/#diagnostic-tests

That should fix this one! (As an experiment you can tweak some of the error messages and see that it'll fail)

Reply:
works - basically status quo